spec/tictactoe_spec.rb

```diff
@@ -7,7 +7,23 @@
   let (:test_ttt) { TicTacToe.new(game_io, player1, player2) }
 
   it "runs the game loop " do
-    allow(test_ttt.game_loop).to receive(:game_loop) { }
-    test_ttt.game_loop
+    expect(test_ttt.game_loop).to receive(:game_loop).once
+    expect(test_ttt).to receive(:play_again?).and_return(false)
+    test_ttt.play_game
+  end
+
+  it "plays the game twice " do
+    expect(test_ttt.game_loop).to receive(:game_loop).twice
+    allow(test_ttt).to receive(:new_game)
+    expect(test_ttt).to receive(:play_again?).and_return(true, false)
```

Comment:
I switched it to stub the GameIO instead. Will look into changing it to a custom MockIo, now.